model-dmdocument/src/main/java/gov/nasa/pds/model/plugin/DMDocument.java

```diff
@@ -456,9 +454,22 @@ public static void main(String args[]) throws Throwable {
     processArgumentParserNamespacePhase1(argparse4jNamespace);
 
     // first get the environment variables
-    getEnvMap();
-    dataDirPath = lPARENT_DIR + "/Data/";
-
+    String USER_DIR = System.getProperty("user.dir");
+    if (USER_DIR == null) {
+      registerMessage("3>error Environment variable USER_DIR is null");
+      printErrorMessages();
+      System.exit(1);
+    }
+    // 666
+//    System.out.println ("debug DMDocument -1- USER_DIR:" + USER_DIR);
+    USER_DIR = USER_DIR.replace('\\', '/');

```

Comment:
Also, I would rename this variable to use `camelCase`, which is more inline with common Java variable naming best practices. All caps are usually reserved for static environment variables, not local variables.